### ISSUE TITLE
User Documentation

### ISSUE THREAD

**Issue by michib** [michib/nextcloud_twofactor_webauthn#10]
Some user docs with screenshots would be really nice

**Comment by derritter88** [comment]
I agree as I tried to install the app to test it but Nextcloud just says 

> Exception: App "Two-Factor Webauthn" cannot be installed because the following dependencies are not fulfilled: The library gmp is not available.


Althought I have installed such a library.

Will play around to find the right one.
